src/plugins/intel_gna/src/backend/gna_limitations.hpp

```diff
@@ -58,9 +59,41 @@ constexpr uint32_t bytesPerCropElement = 2;
 
 constexpr uint32_t kMemoryPageSize = 4096;
 
+class RuntimeTarget {
```

Comment:
I think name does not suit do the class purpose.

Reply:
do_not_review label - It is not supposed to be reviewed